lib/realtime/api.ex

```diff
@@ -112,23 +124,15 @@ defmodule Realtime.Api do
 
   """
   def update_tenant(%Tenant{} = tenant, attrs) do
-    tenant
-    |> Tenant.changeset(attrs)
-    |> tap(&maybe_trigger_disconnect/1)
-    |> Repo.update()
-  end
+    changeset = Tenant.changeset(tenant, attrs)
+    updated = Repo.update(changeset)
```

Comment:
Do we want to only processed if this was an `{:ok, _}` ?

Reply:
good call